tools/runners/test262-harness.py

```diff
@@ -490,7 +541,12 @@ def __init__(self, suite, name, full_path, strict_mode, command_template, module
         self.strict_mode = strict_mode
         with open(self.full_path) as file_desc:
             self.contents = file_desc.read()
-        test_record = parse_test_record(self.contents, name)
+        if self.suite.es5_1:
+            test_record = parse_test_record_es5_1(self.contents, name)
+            if 'negative' in test_record:
+                test_record['negative'] = None
```

Comment:
> Why does `test_record` need a post-processing after `parse_test_record_es5_1`. Why does the function not deal with negative already?

Because in es5_1,  parse_test_record_es5_1 is a direct copy of test_262.py , I dindn't change the sematic,

This pos-processing is used to compatibility with test262-harness.py